DQM/L1TMonitor/src/L1TRPCTF.cc

```diff
@@ -32,15 +32,6 @@ L1TRPCTF::L1TRPCTF(const ParameterSet& ps)
 
   if(verbose_) cout << "L1TRPCTF: constructor...." << endl;
 
-  outputFile_ = ps.getUntrackedParameter<string>("outputFile", "");
```

Comment:
If this parameter is not used anymore it should also be removed from the config files.